app/controllers/support/devices/schools_controller.rb

```diff
@@ -4,6 +4,11 @@ def show
     @users = @school.users
   end
 
+  def confirm_invitation
+    @school = School.find_by!(urn: params[:school_urn])
+    @school_contact = @school.preorder_information&.school_contact
```

Comment:
if school contact is `nil` then `@school_contact` will be set to `nil`. This will cause an error in the view eg when `@school_contact.title.present` is called. I don't know if this will ever be nil though as I lack some knowledge there?

Reply:
Thanks. Fixed in 8892914